extension/src/popup/views/__tests__/Swap.test.tsx

```diff
@@ -108,6 +108,28 @@ jest.mock("popup/helpers/horizonGetBestPath", () => ({
 
 jest.mock("lodash/debounce", () => jest.fn((fn) => fn));
 
+jest.mock("stellar-sdk", () => {
```

Comment:
This might have been the culprit for flaky tests. I missed that it was called somewhere down the line. :see_no_evil:

Reply:
🎉 🎉 🎉